Markdown/Code Block Syntaxes.sublime-completions

```diff
@@ -143,20 +155,50 @@
 			"kind": ["markup", "s", "Syntax"],
 			"details": "Specifies <code>DOS batch</code> code highlighting"
 		},
+		{
+			"trigger": "batch",
+			"annotation": "DOS batch",
+			"kind": ["markup", "s", "Syntax"],
+			"details": "Specifies <code>DOS batch</code> code highlighting"
+		},
+		{
+			"trigger": "batchfile",
+			"annotation": "DOS batch",
+			"kind": ["markup", "s", "Syntax"],
+			"details": "Specifies <code>DOS batch</code> code highlighting"
+		},
 		{
 			"trigger": "cmd",
 			"annotation": "DOS batch",
 			"kind": ["markup", "s", "Syntax"],
 			"details": "Specifies <code>DOS batch</code> code highlighting"
 		},
 		{
-			"trigger": "dos",
+			"trigger": "cmd.exe",
```

Comment:
Maybe we should provide completions only for a sane subset of supported tags.

Reply:
Is this better? This matches Batchfile entry in Linguist:

```yml
Batchfile:
  type: programming
  aliases:
  - bat
  - batch
  - dosbatch
  - winbatch
  extensions:
  - ".bat"
  - ".cmd"
```